ansible.cfg

```diff
@@ -7,4 +7,4 @@ hash_behaviour = replace
 [ssh_connection]
 control_path = %(directory)s/%%C
 pipelining = True
-ssh_args = -o PreferredAuthentications=publickey -o ControlMaster=auto -o ControlPersist=5m
+ssh_args = -o PreferredAuthentications=publickey -o ControlMaster=auto -o ControlPersist=5m -o UserKnownHostsFile=/dev/null -o IdentitiesOnly=yes -o PasswordAuthentication=no -o ConnectTimeout=320
```

Comment:
i don't think we should generally propose `UserKnownHostsFile=/dev/null` the rest looks kinda okish to me. But if we want to solve CI issue we should better do this in the CI itself